products/pdfmorph_license.txt

```diff
@@ -0,0 +1,36 @@
+This program is part of the DiffPy open-source project at Columbia
```

Comment:
Is it possible to connect directly to the license file in the package top level directory?  This would be more robust/maintainable than having two license files hanging around.  What if they are different?  Which one is the correct one?

Reply:
> Is it possible to connect directly to the license file in the package top level directory? This would be more robust/maintainable than having two license files hanging around. What if they are different?

For diffpy-cmi packages I use a dedicated `license` branch at
https://github.com/diffpy/diffpy-release/tree/license.
I make any text updates in the "license" branch and merge it to each package just before release, e.g., [here](https://github.com/diffpy/diffpy.structure/commit/ee4ce3c484eff4db29d29e5fc5f9aaf85929d38d).

Perhaps we could add a new `license-pdfmorph` branch to the "diffpy-release" repo and keep on merging it to this webpage sources and to the "diffpy.pdfmorph" master as well.  If license files come from a common upstream branch, they will stay in sync.

Reply:
is this something that would be top-priority upon release?